airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -476,6 +477,7 @@ def upload(
         :param timeout: Request timeout in seconds.
         :param num_max_attempts: Number of attempts to try to upload the file.
         :param metadata: The metadata to be uploaded with the file.
+        :param cache_control: Cache-Control metadata field
```

Comment:
```suggestion
        :param cache_control: Cache-Control metadata field.
```

Reply:
Alternatively, we could remove the periods from all the above params to keep it consistent. WDYT?